common/culture/traditions/00_combat_traditions.txt

```diff
@@ -1025,16 +1063,11 @@ tradition_strength_in_numbers = {
 	}
 	
 	ai_will_do = {
-		value = 5 # The AI shouldn't really pick this very often
+		# Warcraft
+		value = 5
 		if = {
-			limit = {
-				OR = {
-					culture_has_heavy_infantry_maa = yes
-					culture_has_pikemen_maa = yes
-					culture_has_heavy_cavalry_maa = yes
-				}
-			}
-			multiply = 0
```

Comment:
It was removed because they were being used by inactive regional MaA, not the standard MaA. I've restored them and branched off a [new issue](#1001) to adapt the trigger for GoA2 MaAs.